src/poetry/console/commands/install.py

```diff
@@ -191,8 +191,8 @@ def handle(self) -> int:
                 "If you do not want to install the current project"
                 " use <c1>--no-root</c1>.\n"
                 "If you want to use Poetry only for dependency management"
-                " but not for packaging, you can set the operating mode to "
-                '"non-package" in your pyproject.toml file.\n'
+                " but not for packaging, you can use non-package mode by setting "
+                '"package-mode = false" in your pyproject.toml file.\n'
```

Comment:
```suggestion
                " but not for packaging, you can disable package mode by setting"
                " <c1>package-mode = false</> in your pyproject.toml file.\n"
```

Bit nit picky, but might be better to stick to the terminology in the docs "disable package mode" instead of "use non-package mode".